main.go

```diff
@@ -39,12 +42,22 @@ type data struct {
 
 func value(output tfjson.StateOutput) string {
 	if output.Sensitive {
-		return "sensitive; redacted"
+		return sensitive
 	}
 
 	return fmt.Sprintf("%v", output.Value)
 }
 
+func prettyPrintValue(output tfjson.StateOutput) template.HTML {
+	if output.Sensitive {
+		return template.HTML(sensitive)
+	}
+
+	pretty, _ := json.MarshalIndent(output.Value, "", "  ")
```

Comment:
I'm deliberately ignoring the `error` returned by `json.MarshalIndent`; is that acceptable?

Reply:
I can't think of a case where this would fail, but I'd probably exit with an error message in case it ever does.